lib/active_remote/cached.rb

```diff
@@ -24,6 +24,10 @@ def self.cache(cache_provider = nil)
       @cache_provider
     end
 
+    def self.cache_options
+      ::ActiveRemote::Cached.default_options.except(:cache_error_proc, :handle_cache_error)
```

Comment:
Should this be a whitelist instead? so we are only sending the options we know about?

Reply:
Good call